src/amber/support/dsls.cr

```diff
@@ -23,4 +23,24 @@ module Amber::Support::DSL
 
     {% end %}
   end
+
+  module ControllerActions
+    macro render_both(filename, layout)
+      content = render_template("{{filename.id}}")
+      render_template("{{layout.id}}")
+    end
+
+    # helper to render a template.  The view name is relative to `src/views` directory.
+    macro render_template(filename, *args)
```

Comment:
A nice comment explaining the usage of this method would be helpful since it receives an x amount of arguments.